tier4/checklist.md

```diff
@@ -0,0 +1,482 @@
+        repository's intended purpose, subject area, community, or language.*
+
+
+**Include in Home Page:**
+- [ ] Releases
+- [ ] Packages
+- [ ] Deployments
+
+
+#### Results
+*Insert Review Here*
+
+
+
+
+### Review OpenSSF Scorecard 
+| Checks                   | Description & Condition                                                                                          | Risk   | Min | Score |
+|--------------------------|------------------------------------------------------------------------------------------------------------------|--------|-----|-------|
+| [Dangerous-Workflow](https://github.com/ossf/scorecard/blob/main/docs/checks.md#dangerous-workflow)       | Does the project avoid dangerous coding patterns in GitHub Actions? (e.g. Untrusted Code Checkout, Script Injection with Untrusted Context Variables) | Critical | 10  | ✅ ❌  |
+| [Dependency-Update-Tool](https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool)   | Does the project use tools to help update its dependencies e.g. Dependabot, RenovateBot?                        | High   | 10  |       |
+| [Token-Permissions](https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions)        | Does the project declare GitHub workflow tokens as read only?                                                   | High   | 9   |       |
+| [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection)        | Does the project use Branch Protection?                                                                          | High   | 6   |       |
+| [Code-Review](https://github.com/ossf/scorecard/blob/main/docs/checks.md#code-review)              | Does the project require code review before code is merged?                                                    | High   | 10  |       |
+| [Binary-Artifacts](https://github.com/ossf/scorecard/blob/main/docs/checks.md#binary-artifacts)         | Is the project free of checked-in binaries?                                                                     | High   | 10  |       |
+| [Maintained](https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained)               | Is the project maintained?                                                                                      | High   | 10  |       |
+| [Vulnerabilities](https://github.com/ossf/scorecard/blob/main/docs/checks.md#vulnerabilities)          | Does the project have unfixed vulnerabilities? Uses the [OSV service](https://osv.dev/).                                           | High   | 8   |       |
+| [Fuzzing](https://github.com/ossf/scorecard/blob/main/docs/checks.md#fuzzing)           | Does the project use fuzzing tools, e.g. OSS-Fuzz? (For projects that accept user input)                | Medium | 10  |       |
+| Static Code Analysis Tools| Does the project use static code analysis tools, e.g. CodeQL, LGTM, SonarCloud?                                 | Medium | 10  |       |
```

Comment:
Add this link to the "Static Code Analysis Tools": https://github.com/ossf/scorecard/blob/main/docs/checks.md#sast